castle/static/castle/jumbotron.css

```diff
@@ -39,6 +39,18 @@ div#news ul.meta li {
 	margin-right: 10px;
 }
 
+div.col-md-4 span.byline {
+	font-size: 12px;
+}
+
+span.byline {
+	font-style: italic;
```

Comment:
This is even by my own admission the wrong place to give feedback, but there aren't any better alternatives. 

I don't like this change. Italics are used to _give emphasis_. Formatting the by-line in italics causes visual clutter, since the emphasis is undue. This is aggravated by the combination change of reduced font size and italics, producing an area of the page that seems important (but isn't, I don't want to read the by-line first on a page) and is harder to read than usual.

As a side note, I like the reduced font size as it makes the screen space more compact and is an aesthetically pleasing variation. The italics, however, reduce usability,